pkg/registry/registry.go

```diff
@@ -0,0 +1,63 @@
+package registry
+
+import (
+	"encoding/json"
+	"gz-beat-shipper/pkg/filehandler"
+	"log"
+)
+
+type Registry struct {
+	ParsedFiles []File `json:"ParsedFile"`
+}
+
+type File struct {
+	Name string `json:"name"`
+}
+
+// Append File struct to Registry Struct with ParsedFiles slice
+func (r *Registry) AppendFileToRegistry(f File) {
+	r.ParsedFiles = append(r.ParsedFiles, f)
+}
+
+// Convert Registry Struct to bytes in JSON
+func ConvertRegistryToJSON(registry Registry) []byte {
+	jsonRegistry, err := json.Marshal(registry)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return jsonRegistry
+}
```

Comment:
This function is good as is, but you could make it more Go-like:
* I find it strange that json data is returned in binary format. It is text, let's take that advantage and return it as a string.
* I see no reason to not make this a method instead of a function.

You could rewrite this then as: `func (r *Registry) JSON() string {...}`.
* GoLang likes to omit 'To' or 'Get' prefixes on functions.

Reply:
Fixed.